cmd/snap-confine/mount-support.c

```diff
@@ -177,27 +177,36 @@ static void sc_setup_mount_profiles(const char *snap_name)
 {
 	debug("%s: %s", __FUNCTION__, snap_name);
 
-	FILE *f __attribute__ ((cleanup(sc_cleanup_endmntent))) = NULL;
-	const char *mount_profile_dir = "/var/lib/snapd/mount";
-
+	FILE *desired __attribute__ ((cleanup(sc_cleanup_endmntent))) = NULL;
+	FILE *current __attribute__ ((cleanup(sc_cleanup_endmntent))) = NULL;
```

Comment:
I'm open to change the naming to something else (applied is fine) but to avoid confusion I'd prefer to consistently rename it in all the places after it is merged.